dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNStaticVarCompensatorInterfaceIIDM.cpp

```diff
@@ -49,11 +49,8 @@ staticVarCompensatorIIDM_(svc) {
   stateVariables_[VAR_STATE] = StateVariable("state", StateVariable::INT);  // connectionState
   stateVariables_[VAR_REGULATINGMODE] = StateVariable("regulatingMode", StateVariable::INT);  // regulatingMode
 
-  string libPath = getMandatoryEnvVar("DYNAWO_IIDM_EXTENSION");
-  if (!exists(libPath))
-    throw DYNError(Error::STATIC_DATA, WrongExtensionPath, libPath);
-
-  auto extensionDef = IIDMExtensions::getExtension<StaticVarCompensatorInterfaceIIDMExtension>(libPath);
+  auto libPath = IIDMExtensions::findLibraryPath();
+  auto extensionDef = IIDMExtensions::getExtension<StaticVarCompensatorInterfaceIIDMExtension>(libPath.generic_string());
```

Comment:
what happens if the libPath is empty?

Reply:
In that case, the library could not be loaded and we will return the default extension definition that does nothing (see getExtension function)